File: Assignment_3/program2.c
#include<stdio.h>
void PrintEven(int iNo)
{
    int i=0;
   
   if(iNo<=0)
   {
    iNo=-iNo;
   }
  for(i=1;i<=iNo;i++)
  {
    if(iNo%i==0&& iNo%2==0)
    {
        printf("%d\t",i);
    }
  
  }
}
int main()
{
    int iValue=0;
    

    printf("enter number:");
    scanf("%d",&iValue);
    
    PrintEven(iValue);
    return 0;
}

File: Assignment_18/program2.java
import java.util.Scanner;
class LogicX
{
    void printEvenNumber(int iNo)
    {
        int iCnt=0;

        for(iCnt=0;iCnt<iNo;iCnt++)
        {
            if(iCnt%2==0)
            {
                System.out.println(iCnt);
            }
        }
    }
}
class program2
{
    public static void main(String A[])
    {
        int iValue=0;
        
        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter the number to print Even numbers upto N :");
        iValue=sobj.nextInt();
       
        LogicX lobj=new LogicX();

        lobj.printEvenNumber(iValue);


        
    }
}